### DISCUSSION TITLE
Global text-align:left overriding align=center in rich text fields in admin CP

### DISCUSSION THREAD

**Discussion by ldstevens** [craftcms/cms#8379] [Ideas]
Running into this tiny CSS issue on Craft 2.6...

I have HTML content that's pasted into a rich text field by my client (long story), including years of existing content. 

The HTML content includes tables with `<td align="center">` for captions (it's wysiwyg generated :/ ). 

In the Craft CP however the following rule:
`body.ltr th, body.ltr td {text-align: left}` 
...is being applied, which is overriding  align=center. 

Ideally Redactor wrapper classes would reset these text-align styles to `none` (which is invalid), but having tried this out, it doesn't really make sense. Setting text-align for anything automatically overrides the align param, and things like `initial` default to left, as does `inherit` in this case. So it looks like we have to set _something_. 

Given that is (I think) the case, can:
``.redactor-editor [align="center"] {text-align: center;} ``
be added to the Redactor CSS?

Thanks!

**Comment by carlcs** [comment]
There are plugins available for loading custom CSS for Control Panel views.

https://github.com/lindseydiloreto/craft-cpcss
https://github.com/carlcs/craft-redactorinlinestyles

**Comment by ldstevens** [comment]
@carlcs good call, thanks for the tip!
